exec/csnex/csnex.cs

```diff
@@ -1,6 +1,14 @@
 using System;
 using System.Diagnostics;
+using System.Reflection;
 
+// Fixes MSBuild warning CA1016: Microsoft.Design : Add an AssemblyVersion attribute to 'csnex.exe'.
+// C# Versioning is as follows: <major version>.<minor version>.<build number>.<revision>
+// Major:     0 -- Version of csnex
+// Minor:    50 -- Minor version of csnex
+// Build:     0 -- Build # of csnex
+// Revision:  3 -- Version of Neon Bytecode this executor can execute.
```

Comment:
I don't think it's a good idea to include this here. The bytecode version does not need to be part of the *software* version.